### ISSUE TITLE
openssh has clock-y runtime errors on El Cap with Xcode 8

### ISSUE THREAD

**Issue by chdiza** [Homebrew/homebrew-dupes#656]
`openssh` builds fine on El Cap with Xcode 8, but barfs at runtime:
```
dyld: lazy symbol binding failed: Symbol not found: _clock_gettime
  Referenced from: /usr/local/Cellar/openssh/7.3p1/bin/ssh
  Expected in: /usr/lib/libSystem.B.dylib

dyld: Symbol not found: _clock_gettime
  Referenced from: /usr/local/Cellar/openssh/7.3p1/bin/ssh
  Expected in: /usr/lib/libSystem.B.dylib
```

I really hate Xcode 8.  Everything works as expected on El Cap with Xcode 731.

**Comment by ilovezfs** [comment]
>I really hate Xcode 8. Everything works as expected on El Cap with Xcode 731.

If you upgrade to 10.12, these problems will go away for you.

**Comment by ilovezfs** [comment]
This one is actually already fixed due to https://github.com/Homebrew/brew/commit/88a1b2853a2206a43f957ea71657c1b6d9bc37bb

**Comment by chdiza** [comment]
> If you upgrade to 10.12, these problems will go away for you.

If I upgrade to 10.12, they will be replaced by problems that are even worse.

> This one is actually already fixed due to Homebrew/brew@88a1b28

Yes, works here, thanks.

**Comment by ilovezfs** [comment]
>If I upgrade to 10.12, they will be replaced by problems that are even worse.

Quite possible. Just making sure you're aware the clock stuff is specific to 10.11+Xcode 8 not Xcode 8 per se.

>Yes, works here, thanks.

You're quite welcome!
